# [anvio](https://hpc.nih.gov/apps/anvio.html)

Create the following symbolic links:
```
anvi-display-structure -> anvio.sh
anvi-script-gen-help-pages -> anvio.sh
anvi-meta-pan-genome -> anvio.sh
anvi-export-splits-and-coverages -> anvio.sh
anvi-setup-kegg-kofams -> anvio.sh
anvi-script-transpose-matrix -> anvio.sh
anvi-gen-fixation-index-matrix -> anvio.sh
anvi-compute-gene-cluster-homogeneity -> anvio.sh
anvi-import-collection -> anvio.sh
anvi-dereplicate-genomes -> anvio.sh
anvi-split -> anvio.sh
anvi-get-sequences-for-gene-calls -> anvio.sh
anvi-profile -> anvio.sh
anvi-script-gen-distribution-of-genes-in-a-bin -> anvio.sh
anvi-db-info -> anvio.sh
anvi-display-pan -> anvio.sh
anvi-show-collections-and-bins -> anvio.sh
anvi-script-filter-fasta-by-blast -> anvio.sh
anvi-script-snvs-to-interactive -> anvio.sh
anvi-script-get-primer-matches -> anvio.sh
anvi-gen-variability-network -> anvio.sh
anvi-script-run-eggnog-mapper -> anvio.sh
anvi-script-augustus-output-to-external-gene-calls -> anvio.sh
anvi-script-gen-pseudo-paired-reads-from-fastq -> anvio.sh
anvi-cluster-contigs -> anvio.sh
anvi-export-table -> anvio.sh
anvi-export-locus -> anvio.sh
anvi-script-gen-hmm-hits-matrix-across-genomes -> anvio.sh
anvi-update-db-description -> anvio.sh
anvi-script-estimate-genome-size -> anvio.sh
anvi-script-pfam-accessions-to-hmms-directory -> anvio.sh
anvi-estimate-metabolism -> anvio.sh
anvi-export-splits-taxonomy -> anvio.sh
anvi-init-bam -> anvio.sh
anvi-setup-pfams -> anvio.sh
anvi-script-tabulate -> anvio.sh
anvi-export-gene-coverage-and-detection -> anvio.sh
anvi-script-gen-CPR-classifier -> anvio.sh
anvi-estimate-scg-taxonomy -> anvio.sh
anvi-compute-ani -> anvio.sh
anvi-gen-variability-profile -> anvio.sh
anvi-show-misc-data -> anvio.sh
anvi-setup-trna-taxonomy -> anvio.sh
anvi-search-functions -> anvio.sh
anvi-trnaseq -> anvio.sh
anvi-refine -> anvio.sh
anvi-export-contigs -> anvio.sh
anvi-script-process-genbank-metadata -> anvio.sh
anvi-import-functions -> anvio.sh
anvi-experimental-organization -> anvio.sh
anvi-run-ncbi-cogs -> anvio.sh
anvi-setup-ncbi-cogs -> anvio.sh
anvi-export-functions -> anvio.sh
anvi-import-items-order -> anvio.sh
anvi-get-codon-frequencies -> anvio.sh
anvi-help -> anvio.sh
anvi-merge -> anvio.sh
anvi-script-gen_stats_for_single_copy_genes.py -> anvio.sh
anvi-import-taxonomy-for-genes -> anvio.sh
anvi-script-get-collection-info -> anvio.sh
anvi-compute-genome-similarity -> anvio.sh
anvi-import-taxonomy-for-layers -> anvio.sh
anvi-oligotype-linkmers -> anvio.sh
anvi-run-workflow -> anvio.sh
anvi-export-items-order -> anvio.sh
anvi-run-trna-taxonomy -> anvio.sh
anvi-gen-structure-database -> anvio.sh
anvi-gen-variability-matrix -> anvio.sh
anvi-script-gen-scg-domain-classifier -> anvio.sh
anvi-import-state -> anvio.sh
anvi-pan-genome -> anvio.sh
anvi-run-kegg-kofams -> anvio.sh
anvi-push -> anvio.sh
anvi-merge-bins -> anvio.sh
anvi-script-process-genbank -> anvio.sh
anvi-get-aa-counts -> anvio.sh
anvi-script-visualize-split-coverages -> anvio.sh
anvi-inspect -> anvio.sh
anvi-compute-completeness -> anvio.sh
anvi-estimate-genome-completeness -> anvio.sh
anvi-delete-collection -> anvio.sh
anvi-self-test -> anvio.sh
anvi-gen-genomes-storage -> anvio.sh
anvi-script-gen_stats_for_single_copy_genes.R -> anvio.sh
anvi-setup-interacdome -> anvio.sh
anvi-export-gene-calls -> anvio.sh
anvi-run-pfams -> anvio.sh
anvi-run-scg-taxonomy -> anvio.sh
anvi-matrix-to-newick -> anvio.sh
anvi-script-variability-to-vcf -> anvio.sh
anvi-script-enrichment-stats -> anvio.sh
anvi-delete-hmms -> anvio.sh
anvi-get-short-reads-mapping-to-a-gene -> anvio.sh
anvi-update-structure-database -> anvio.sh
anvi-export-state -> anvio.sh
anvi-mcg-classifier -> anvio.sh
anvi-get-sequences-for-hmm-hits -> anvio.sh
anvi-display-metabolism -> anvio.sh
anvi-report-linkmers -> anvio.sh
anvi-run-interacdome -> anvio.sh
anvi-script-reformat-fasta -> anvio.sh
anvi-run-hmms -> anvio.sh
anvi-gen-gene-consensus-sequences -> anvio.sh
anvi-gen-network -> anvio.sh
anvi-script-fix-homopolymer-indels -> anvio.sh
anvi-script-gen-short-reads -> anvio.sh
anvi-export-collection -> anvio.sh
anvi-delete-misc-data -> anvio.sh
anvi-script-add-default-collection -> anvio.sh
anvi-script-checkm-tree-to-interactive -> anvio.sh
anvi-export-misc-data -> anvio.sh
anvi-gen-phylogenomic-tree -> anvio.sh
anvi-estimate-trna-taxonomy -> anvio.sh
anvi-scan-trnas -> anvio.sh
anvi-delete-state -> anvio.sh
anvi-analyze-synteny -> anvio.sh
anvi-script-gen-programs-vignette -> anvio.sh
anvi-gen-contigs-database -> anvio.sh
anvi-get-split-coverages -> anvio.sh
anvi-import-misc-data -> anvio.sh
anvi-interactive -> anvio.sh
anvi-setup-pdb-database -> anvio.sh
anvi-display-contigs-stats -> anvio.sh
anvi-gen-gene-level-stats-databases -> anvio.sh
anvi-estimate-genome-taxonomy -> anvio.sh
anvi-script-get-hmm-hits-per-gene-call -> anvio.sh
anvi-script-calculate-pn-ps-ratio -> anvio.sh
anvi-script-predict-CPR-genomes -> anvio.sh
anvi-convert-trnaseq-database -> anvio.sh
anvi-get-short-reads-from-bam -> anvio.sh
anvi-script-get-coverage-from-bam -> anvio.sh
anvi-migrate -> anvio.sh
anvi-get-enriched-functions-per-pan-group -> anvio.sh
anvi-script-gen_stats_for_single_copy_genes.sh -> anvio.sh
anvi-summarize -> anvio.sh
anvi-get-sequences-for-gene-clusters -> anvio.sh
anvi-compute-functional-enrichment -> anvio.sh
anvi-rename-bins -> anvio.sh
anvi-script-merge-collections -> anvio.sh
anvi-upgrade -> anvio.sh
anvi-setup-scg-taxonomy -> anvio.sh
anvi-export-structures -> anvio.sh
anvi-script-compute-ani-for-fasta -> anvio.sh
anvi-script-gen-programs-network -> anvio.sh
```
